slash/react/src/Link/LinkAnchor.tsx

```diff
@@ -21,11 +23,14 @@ const LinkAnchor = forwardRef<HTMLAnchorElement, LinkComponentProps>(
       children,
       rightIcon,
       disabled,
+      variant,
       ...restProps
     }: LinkComponentProps,
     ref,
   ) => {
-    const finalClassName = classnames(linkClassName, className);
+    const finalClassName = classnames(linkClassName, className, {
+      [`${linkClassName}--reverse`]: variant === "reverse",
+    });
```

Comment:
```suggestion
    const finalClassName = classnames(linkClassName, {
      [`${linkClassName}--reverse`]: variant === "reverse", className
    });
```
L'ajout de class css pour l'utilisateur doit toujours être placer après le style du composant